javascript/transports/action_cable.js

```diff
@@ -45,30 +46,30 @@ const connected = () => {
   active = true
   connectionStatusClass()
   emitEvent('stimulus-reflex:connected')
-  emitEvent('stimulus-reflex:action-cable:connected')
-
-  queue.forEach(reflex => subscription.send(reflex))
-  queue = []
+  Object.values(reflexes.queued).forEach(reflex => {
+    subscription.send(reflex.data)
+    dispatchLifecycleEvent(reflex, 'delivered')
+  })
```

Comment:
Can you link to where in the PR these reflexes move stages in the store? I keep getting distracted during review and lose context. I want to grok this aspect of the PR a little better.

Reply:
Man oh man, do I ever empathize with this.

https://github.com/stimulusreflex/stimulus_reflex/blob/544af220c6e8d6e561a3b1f1f081973642bef1c3/javascript/lifecycle.js#L78